recipes/tcmarkers/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set version = "0.0.3" %}
+
+package:
+  name: tcmarkers
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/t/tcmarkers/tcmarkers-{{ version }}.tar.gz
+  sha256: 2e31ee5bae5462b6d23b8902af748fc419792512c5893deb7d7e25ceb16a6385
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools_scm
+    - setuptools_scm_git_archive
+  run:
+    - python >=3.7
+    - matplotlib-base
+    - numpy
+
+test:
+  imports:
+    - tcmarkers
```

```suggestion
test:
  imports:
    - tcmarkers
  commands:
    - pip check
  requires:
    - pip
```
This makes sure that all packages are there.